example/rest-api/art-institute/art-institute.php

```diff
@@ -40,20 +65,28 @@ function register_aic_block(): void {
 			],
 		],
 		'output_schema' => [
-			'is_collection' => false,
-			'path' => '$.data',
+			'is_collection' => true,
```

Comment:
Does `supports_bulk = true` imply `is_collection = true`?

Reply:
I actually should rephrase, I think that's the wrong terminology. Does a block having `supports_bulk = true` on the selection query imply that the get query needs to be `is_collection = true`?

Reply:
>Does a block having supports_bulk = true on the selection query imply that the get query needs to be is_collection = true?

The answer is yes, but I'm not sure if it should be. So great question! 😄 

To use our existing setup, it needs to be `true` so we can handle multiple returned results, otherwise, we only expect a single result to be returned. 

But now I'm curious why we have`is_collection` in the first place. In trunk without multi-select, if I set the collection to `true` for this example, we can still display the single result without issue. 

To me, it seems safer to handle everything as an array so we can either map through one or more results and not need the `is_collection.`

Reply:
> But now I'm curious why we have `is_collection` in the first place.

Short answer: To provide a consistent response shape from the `remote-data` REST endpoint. 

Longer answer involves the fact that JSONPath always returns an array, even for single values:

<img width="693" alt="Screenshot 2025-02-19 at 10 02 20" src="https://github.com/user-attachments/assets/8593f416-874a-471a-8884-b017f048cd08" />

We use `is_collection` when parsing response data to understand if the return value is a single item or, coincidentally a collection with only one member.